maven/bnd-maven-plugin/README.md

```diff
@@ -117,25 +195,57 @@ The `-noextraheaders: true` instruction will prevent Bnd from adding extra manif
 
 This plugin supports a hybrid configuration model where Bnd instructions can come from a bnd file or configuration in the project pom. Inheritance of configuration from parent projects is also supported for this hybrid configuration model. At each project level in the project hierarchy, the configuration can come from a bnd file in the project or from the configuration in the pom with the former taking precedence. This plugin merges the configurations from the parent project with the configuration from the current project. If a parent project does not define a configuration for this plugin, then the configuration, if any, from the `pluginManagement` section for this plugin is used as the configuration from the parent project. This configuration contribution from the `pluginManagement` section for this plugin is evaluated in the context of the current project.
 
-### IMPORTANT NOTE
+### Usage Scenarios
 
-The `maven-jar-plugin` will NOT currently use the data from the generated `MANIFEST.MF` file when using its default configuration. We anticipate a [patch][1] to the JAR plugin that will do this. In the meantime it is necessary to configure the `maven-jar-plugin` plugin as follows:
+The plugin has 6 distinct usage scenarios broken into two groups.
 
-```xml
-<plugin>
-    <groupId>org.apache.maven.plugins</groupId>
-    <artifactId>maven-jar-plugin</artifactId>
-    <configuration>
-        <archive>
-            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
-        </archive>
-    </configuration>
-</plugin>
-```
+#### Group 1: Scenarios based on the `jar` goal
 
-### Usage Scenarios
+Given executions using the `jar` goal we have the following 3 cases:
+
+1. The common case where very little configuration is required; inputs and outputs are based on defaults. Bnd performs it's analysis and enhances the jar with OSGi metadata obtained through introspection of classes, resources, dependencies, and [OSGi bundle annotations](https://osgi.org/specification/osgi.core/7.0.0/framework.api.html#org.osgi.annotation.bundle):
+
+   ```xml
+   <plugin>
+       <groupId>biz.aQute.bnd</groupId>
+       <artifactId>bnd-maven-plugin</artifactId>
```

Comment:
I was trying to leverage the notion that `<extensions>true</extensions>` would be applied only once in pluginManagement where we defined the default execution because once a plugin applies `<extensions>true</extensions>` it's effective globally.

Reply:
Yes but this is the readme and should probably be explicit. Some repos have only one project and no need to pluginManagement

Reply:
:+1:

Reply:
But we could make the same argument for all the `bnd-process` examples where there is no execution specified.

Reply:
true :-)

Reply:
We can go with what is here. We can always improve it later if people get confused.